### PR TITLE
Feat/resource name

### PR DESCRIPTION
**Pull request by fabien0102**
### Summary

* A small `ResourceName` component
* A fix for pagination

### CODE REVIEW

**Comment by fabien0102** [comment]
[ResourceName component](https://trello.com/c/5JT0t5uQ/278-resourcename-component)

File: packages/components/src/ResourceName/ResourceName.tsx [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from "react-emotion"
+
+export const ResourceName = styled("span")`
+  font-family: monospace;
```
Comment (micha-f) [inline packages/components/src/ResourceName/ResourceName.tsx]: Are we not using constants.font.code?

Comment (fabien0102) [reply packages/components/src/ResourceName/ResourceName.tsx]: Sure, my bad, I'm probably a bit tired

Comment (micha-f) [reply packages/components/src/ResourceName/ResourceName.tsx]: Drink more ☕️  
😆

File: packages/components/src/ResourceName/ResourceName.tsx [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import styled from "react-emotion"
+import { OperationalStyleConstants } from "../utils/constants"
+
+export const ResourceName = styled("span")`
+  font-family: ${({ theme }: { theme: OperationalStyleConstants }) => theme.font.family.code};
+  color: #333333;
```
Comment (TejasQ) [inline packages/components/src/ResourceName/ResourceName.tsx]: Is there a constant color for this?

Comment (micha-f) [reply packages/components/src/ResourceName/ResourceName.tsx]: `textColors.dark`

File: packages/components/src/Paginator/Paginator.tsx [hunk 3]
```diff
@@ -83,6 +83,7 @@ const getRange = ({ page, itemCount, itemsPerPage }: Props) => {
 
 const Container = styled("div")({
   label: "paginator",
+  marginTop: 20,
```
Comment (TejasQ) [inline packages/components/src/Paginator/Paginator.tsx]: `theme.space.element` I believe.

**Pull request merged**